### DISCUSSION TITLE
Better guide for SSPI?

### DISCUSSION THREAD

**Discussion by fjf2002** [dbeaver/dbeaver#16097] [Ideas]
Hi!

I did not find any hints on how to use SSPI with the postgres driver. I found out myself:

https://fjf2002.wordpress.com/2018/11/01/dbeaver-postgresql-sspi-authentication/

Perhaps you could document/support that better for other users. Thanks!

Sincerely,
fjf2002

**Comment by tolemac** [comment]
I get this exception:

Something unusual has occurred to cause the driver to fail. Please report this exception.
  com.sun.jna.platform.win32.Sspi$SecBufferDesc.<init>(I[B)V


Any way to connect postgresql using sspi?
